addons/survey/views/survey_templates_user_input_session.xml

```diff
@@ -107,14 +107,14 @@
                     </div>
                 </div>
             </div>
-            <div class="o_container_small px-4 pb-3 pt96 d-flex flex-column o_survey_session_manage_container">
```

Comment:
Hello @aath-odoo :slightly_smiling_face: 
2 last picky comments :smile: 

There's no need to remove the `flex-column` here, you can keep it :slightly_smiling_face: 
Also, could you update your commit message (and PR description) for the Issue 1 ? :slightly_smiling_face: 

Thanks!